cordova-sdk/plugin.xml

```diff
@@ -1,32 +1,35 @@
 <?xml version='1.0' encoding='utf-8'?>
-<plugin id="optimove-cordova-sdk" version="2.1.0" xmlns="http://apache.org/cordova/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android">
+<plugin id="optimove-cordova-sdk" version="2.2.0" xmlns="http://apache.org/cordova/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android">
     <name>OptimoveSDKPlugin</name>
     <js-module name="OptimoveCore" src="www/OptimoveCore.js"/>
     <js-module name="Optimove" src="www/Optimove.js">
         <clobbers target="window.Optimove" />
     </js-module>
     <hook type="before_prepare" src="scripts/inject-config.js" />
+
     <platform name="ios">
-        <resource-file src="scripts/optimove.plist" target="optimove.plist" />
-        <config-file target="config.xml" parent="/*">
+        <config-file parent="/*" target="config.xml">
             <feature name="OptimoveSDKPlugin">
                 <param name="ios-package" value="Optimove_Cordova" />
             </feature>
+            <preference name="SwiftVersion" value="5" />
+            <preference name="deployment-target" value="12.0" />
         </config-file>
+        <resource-file src="scripts/optimove.plist" target="optimove.plist" />
         <source-file src="src/ios/OptimoveSDKPlugin.swift" />
         <header-file src="src/ios/InitializePluginManager.h" />
         <source-file src="src/ios/InitializePluginManager.m" />
         <header-file src="src/ios/Bridging-Header.h" type="BridgingHeader" />
         <podspec>
             <config>
-                <source url="https://github.com/CocoaPods/Specs.git"/>
+                <source url="https://cdn.cocoapods.org/"/>
             </config>
             <pods>
-                <pod name="OptimoveCore" spec="5.1.1" />
-                <pod name="OptimoveSDK" spec="5.1.1" />
+                <pod name="OptimoveSDK" spec="~> 5" />
```

Comment:
Optimove SDK has an inner dependency on Optimove Core an will be installed in bulk.

Reply:
i think intention in hybrid sdks was to pin to a specific base sdk version. it means
- bugs in base sdks dont propagate to each hybrid
- easier to relate issues to a specific version

Reply:
Ok. Let's pin the latest known version